src/pyhf/tensor/pytorch_backend.py

```diff
@@ -222,6 +227,7 @@ def simple_broadcast(self, *args):
             list of Tensors: The sequence broadcast together.
         """
 
+        args = [arg.view(1) if not self.shape(arg) else arg for arg in args]
```

Comment:
This one is interesting to me. I suppose you're expecting `tb.broadcast(1, [2,2,2,2])` to return `[1,1,1,1], [2,2,2,2]` ? I'm wondering if this is a desired behavior. I think we do, but just need clarification, as I presumed the PR was about _outputs_ of things, rather than inputs.

Reply:
Yeah

```python
>>> import pyhf
>>> pyhf.tensorlib.simple_broadcast(1, [2,2,2,2])
[array([1, 1, 1, 1]), array([2, 2, 2, 2])]
```

This is actually the behavior of all the native backends' version of broadcast, so I figures we should allow for `()` shape tensors to work here as well to match patterns and to ensure that this works.